locale/content.json

```diff
@@ -238,7 +238,7 @@
         "England": {
           "desc": "Benefit appeals helpline (England and Wales)",
           "phone": "Telephone: 0300 123 1142",
-          "availability": "Monday to Friday, 8am to 6pm",
+          "availability": "Monday to Friday, 8am to 5pm",
           "availability2": "Saturday, 8am to 2pm",
```

Comment:
Are you not removing Saturday like in:
 https://github.com/hmcts/sscs-submit-your-appeal/pull/1320/files

Reply:
Good Spot cheers